admin/tabs/activity/tracking/ActivityModel.js

```diff
@@ -208,10 +207,13 @@ export class ActivityModel {
 
     paramsReaction() {
         return {
-            track: () => this.getParams(),
+            track: () => {
+                const {startDate, endDate, username, msg, category, device, browser} = this.formModel.values;
+                return [startDate, endDate, username, msg, category, device, browser];
+            },
             run: () => this.loadAsync(),
-            equals: comparer.structural,
-            fireImmediately: true
+            fireImmediately: true,
+            delay: 1
```

Comment:
Here because the buttons that adjust the date fields can change both and cause a double network call.

Reply:
we want to use debounce, not delay I think